crates/viewer/re_selection_panel/src/selection_panel.rs

```diff
@@ -1322,4 +1327,156 @@ mod tests {
         harness.run();
         harness.snapshot("selection_panel_component_hybrid_overwrite");
     }
+
+    #[test]
+    fn selection_panel_view_snapshot() {
```

Comment:
I think I would have made those three all call the same parameterized function, but this is fine too I figure; the boilerplate isn't that crazy